native/c/zowex.cpp

```diff
@@ -812,7 +812,7 @@ int handle_job_submit_jcl(ZCLIResult result)
   raw_bytes.clear();
 
   ZEncode encoding_opts = {0};
-  const auto encoding_prepared = result.get_option("--encoding")->is_found() && zut_prepare_encoding(result.get_option("--encoding")->get_value(), &encoding_opts);
+  const auto encoding_prepared = result.get_option("--encoding") != nullptr && result.get_option("--encoding")->is_found() && zut_prepare_encoding(result.get_option("--encoding")->get_value(), &encoding_opts);
```

Comment:
thanks for fixing the S0C4.

Question... why is it that we need to check for the the encoding option no not be null here and nowhere else? 🤯

Reply:
We need to fix it everywhere, I just implemented a quick fix so the command works as expected (as part of my story that was in-flight)

Reply:
Upcoming fix for all scenarios in branch `fix/nullptr/get-option` 😋